topsort-endpoints.yml

```diff
@@ -89,6 +89,34 @@ paths:
           $ref: '#/components/responses/UnauthorizedError'
         400:
           $ref: '#/components/responses/BadRequest'
+      x-readme.code-samples:
```

Comment:
I think this is not going to work because all event types are going to get the same example data